_includes/specials/policy/fr/03-encheres-pour-l-achat-d-un-bloc-d-espace.md

```diff
@@ -0,0 +1,81 @@
+<!--
+  300 to 1000 words
+  put title in main newsletter
+  put links in this file
+  for any subheads use h3 (i.e., ###)
+  illustrations welcome (max width 800px)
+  if uncertain about anything, just do what seems best and harding will edit
+-->
+
+La semaine dernière, nous avons indiqué que les transactions payaient des frais pour l'espace de blocs utilisé plutôt que pour le
+montant transféré, et nous avons établi que les mineurs optimisent leur sélection de transactions pour maximiser les frais perçus.
+Il s'ensuit que seules sont confirmées les transactions qui se trouvent en tête du pool de mémoire lorsqu'un bloc est trouvé.
+Dans ce billet, nous discuterons de stratégies pratiques pour tirer le meilleur parti de nos frais. Supposons que nous disposons
+d'une source décente d'estimations de taux de change---nous parlerons plus en détail de l'estimation des taux de change dans
+l'article de la semaine prochaine.
+
+Lors de l'élaboration des transactions, certaines parties de la transaction sont plus flexibles que d'autres. Chaque transaction
+nécessite les champs d'en-tête, les sorties du destinataire sont déterminées par les paiements effectués, et la plupart des
+transactions nécessitent une sortie de changement. L'expéditeur et le destinataire doivent privilégier les types de sorties
+efficaces en termes d'espace de blocs afin de réduire le coût futur de la dépense de leurs sorties de transaction, mais c'est au
+cours de la [sélection des entrées][topic coin selection] qu'il est le plus possible de modifier la composition et le poids finaux
+de la transaction. Comme les transactions se concurrencent par taux [frais/poids], une transaction plus légère nécessite des frais
+moins élevés pour atteindre le même taux.
+
+Certains portefeuilles, comme le portefeuille Bitcoin Core, essaient de combiner les entrées de manière à éviter d'avoir besoin de
+changer les sorties. Éviter le changement permet d'économiser le poids d'une sortie maintenant, mais aussi le coût futur de la
+dépense de la sortie de changement plus tard. Malheureusement, de telles combinaisons d'entrées ne seront que rarement disponibles,
+à moins que le portefeuille ne dispose d'un grand pool d'UTXO avec une grande variété de montants.
+
+Les types de sortie modernes sont plus efficaces en termes d'espace de blocs que les types de sortie plus anciens. Par exemple,
+dépenser une entrée P2TR coûte moins de 2/5e du poids d'une entrée P2PKH (essayez-le avec notre [calculateur de taille de transaction
```

Comment:
Took me way too long to find this as the issue

```suggestion
dépenser une entrée P2TR coûte moins de 2/5e du poids d'une entrée P2PKH (essayez-le avec notre [calculateur de taille de transaction]
```

Pushed a commit to fix this and also removed what appears to be an extra "(" on the next line